src/DaPigGuy/PiggyFactions/commands/subcommands/TopSubCommand.php

```diff
@@ -41,6 +41,11 @@ public function onBasicRun(CommandSender $sender, array $args): void
             return;
         }
 
+        if ($page > $this->plugin->getConfig()->get("limit.limit-page-top")) {
```

Comment:
Why?

Reply:
because if we do /f top power or member 9999999999999999 the server will crash

Reply:
Your fix creates another problem. Assume that the page limit was set to 10. After the 100th faction, I would not be able to see the factions past unless the server owner manually increases the limit. This is not the desired behavior.

I assume this to be the crashdump in question: https://crash.pmmp.io/view/8706118

In the crashdump, we see that:
```
#0 TopSubCommand(51): array_slice(array[2], float 9.2233720368548E+19, int 10)
#5 pmsrc/src/Server(1422): string[95] f top power 99999999999999999999999999999999999999999999999999999999999999999999)
```

An extremely large number would likely result in an [integer overflow](https://www.php.net/manual/en/language.types.integer.php), thus causing it to become a float.

It should be fine to check for `$page > PHP_INT_MAX`, but I'd rather we silently set the `$page = PHP_INT_MAX` in this case rather than blocking the user. @DaPigGuy what do you think?